src/pages/Templates.tsx

```diff
@@ -26,7 +26,7 @@ type TemplatesState = { subject: string, markdown: string, doc: any } & {
   doc: any
 } & any & { subject: string, markdown: string, doc: any };
 
-class Templates extends Component<{}, TemplatesState> {
+class Templates extends Component<any, TemplatesState> {
```

Comment:
for components with no props you should be able to use the `never` type instead of `any` for the props

Reply:
All the components inside this project have props, which need to get a proper type description. At this point I used the `any` type to make the minimum step in order to fix the typescript errors, as suggested in [TypeScript React Conversion Guide](https://github.com/Microsoft/TypeScript-React-Conversion-Guide#minimum-transition-steps) by Microsoft.